__tests__/app.test.js

```diff
@@ -30,54 +44,61 @@ describe("GET /api/topics", () => {
   });
 });
 
-describe("GET /api/", () => {
-    test("GET:200 - responds with a 200 status code and an object of availiable endpoint names", () => {
-      return request(app)
-        .get("/api/")
-        .expect(200)
-        .then(({_body}) => {
-          expect(_body.endpoints);
-          const endpointsStr = JSON.stringify(_body.endpoints)
-          const endpointsStrRef = JSON.stringify(endpointsRef)
-          expect (endpointsStr).toEqual(endpointsStrRef)
+describe("GET /api/articles", () => {
+  test("GET:200 - responds with a 200 status code and an array of articles with the correct information", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ _body: { articles } }) => {
+        expect(articles.length).toBe(5);
+        articles.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("number");
         });
-    });
+      });
```

Comment:
We want some testing here to show that the articles are sorted in date descending order. You can either amend this test or put a new one in, but please add this 👍

Reply:
Oops I forgot this, Yep I can do that, cheers